Table creation

jasminejanet opened the pull request:
kindly review my table program..

jasminejanet committed:
table format with required fields

jasminejanet committed:
cross-checking

whether the table is working for n number of rows

Sharavanth commented on public/index.html:
```diff
@@ -0,0 +1,20 @@
+  </head>
+  <body>
+    <div id="jas"></div>
+    <script type="text/jsx;harmony=true" src="scripts/type3.js"></script>
```
The file could be named better , eg table.js

Sharavanth replied:
Also unneccesary files could be removed

jasminejanet replied:
ok have done this too...

Sharavanth commented on public/scripts/type3.js:
```diff
@@ -0,0 +1,54 @@
+        return data.map(function(item) {
+          
+            var cells = cols.map(function(colData) {
+
```
Avoid blank lines. Try to conform to coding style guidelines

jasminejanet replied:
Yes.. I have removed now

jasminejanet committed:
adds missing files

Some files were not added in the original commit. So build cannot
progress.. These files are added now.

jasminejanet committed:
close #1
